KERN_ERR is not appropriate for a printk level of a successful operation


--- linux-2.6.30/drivers/hid/hid-wacom.c~	2009-09-04 10:37:20.000000000 -0700
+++ linux-2.6.30/drivers/hid/hid-wacom.c	2009-09-04 10:37:20.000000000 -0700
@@ -244,7 +244,7 @@
 	ret = hid_register_driver(&wacom_driver);
 	if (ret)
 		printk(KERN_ERR "can't register wacom driver\n");
-	printk(KERN_ERR "wacom driver registered\n");
+	printk(KERN_INFO "wacom driver registered\n");
 	return ret;
 }
 
